activerecord/lib/arel/nodes/comment.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Arel # :nodoc: all
+  module Nodes
+    class Comment < Arel::Nodes::Node
+      attr_reader :values
+
+      def initialize(*values)
```

Comment:
I'd not prefer the splat args for the internal node, this allows `Comment.new(single_value)` even the `values` expects multiple values, but that is not necessary flexibility for internal nodes.

Reply:
This has been removed. :+1: